src/components/network-modification-tree.js

```diff
@@ -109,8 +109,12 @@ const NetworkModificationTree = ({ treeModel, studyMapTreeDisplay }) => {
     const onElementClick = useCallback(
         (event, element) => {
             setSelectedNode(element);
-            dispatch(setModificationsDrawerOpen(true));
-            dispatch(selectTreeNode(element.id));
+            dispatch(
+                setModificationsDrawerOpen(
+                    element.type === 'NETWORK_MODIFICATION' ? true : false
```

Comment:
Why restate boolean value and not keep `element.type === 'NETWORK_MODIFICATION'` ?

Reply:
Done